recipe/build-mpi.sh

```diff
@@ -98,6 +98,8 @@ if [[ "$target_platform" == linux-* ]]; then
 fi
 
 ./configure --prefix=$PREFIX \
+            --with-docdir=$PWD/_noinst/doc \
+            --with-mandir=$PWD/_noinst/man \
```

Comment:
A requested some time ago for a `--disable-doc` configure option. https://github.com/pmodels/mpich/issues/6908

```suggestion
            --disable-doc \
```